agent/custom/general_navigation_action.py

```
@@ -1,238 +1,482 @@
+def pair_nametag_tag(boxes):
+    """配对: 返回 (有效 nametag 框列表, 未配对 tag 框列表)
+    boxes: 全部检测框 [(x,y,w,h,cls),...]
+    nametag 框为含 tag 的完整名称条, 配对条件:
+    tag 左边缘位于 nametag 右侧 40% 区域内(且不超出右侧 5px), y 方向有重叠
+    (供外部测试脚本引用, 部署流程已不再使用)"""
+    nametags = [(x, y, w, h) for x, y, w, h, cls in boxes if cls == 0]
+    tags = [(x, y, w, h) for x, y, w, h, cls in boxes if cls == 1]
+    valid = []
+    tag_paired = [False] * len(tags)
+    for nx, ny, nw, nh in nametags:
+        for ti, (tx, ty, tw, th) in enumerate(tags):
```

Ruff (0.16.1)

[warning] Loop control variable `tw` not used within loop body

Rename unused `tw` to `_tw`

(B007)